src/components/showcase/ShowcaseHeader.js

```diff
@@ -19,6 +19,12 @@ import {
 import { useNavigation } from '@rainbow-me/navigation';
 import Routes from '@rainbow-me/routes';
 import { padding } from '@rainbow-me/styles';
+import colors from '@rainbow-me/styles/colors';
```

Comment:
consolidate this line with the line above

`import { colors, padding } from '@rainbow-me/styles`